.one-pipeline.yaml

```diff
@@ -180,8 +180,8 @@ containerize:
      DIGEST="$(skopeo inspect docker://$IMAGE | grep Digest | grep -o 'sha[^\"]*')"
      #ARCH="$(echo $i | grep -o '\(amd64\|s390x\|ppc64le\)$' || echo 'amd64')"
      ARCH=amd64
-     echo "Saving artifact bundle-$i name=$IMAGE digest=$DIGEST manifest=false"
-     save_artifact bundle-$i type=image name="$IMAGE" "digest=$DIGEST" "arch=$ARCH" "manifest=false"
+     echo "Saving artifact bundle-$i name=$IMAGE digest=$DIGEST"
+     save_artifact bundle-$i type=image name="$IMAGE" "digest=$DIGEST" "arch=$ARCH" 
```

Comment:
why is type hard coded to image here?

Reply:
The bundle does not have a manifest image.